code/API_definitions/application-profiles.yaml

```diff
@@ -116,7 +142,7 @@ paths:
           $ref: "#/components/responses/Generic403"
         "404":
           $ref: "#/components/responses/Generic404"
-        "409":
+        "405":
```

Comment:
Typo?

The ref goes still to 409, and 405 is not a valid response code for a defined method. See also https://github.com/camaraproject/Commonalities/issues/285

Reply:
i missed updating the path to point to 405. thanks for your thorough review and catching this Herbert.

Reply:
405 is not a valid response code. have removed it from the yaml and also cleaned up the unused 409 code definition.

Reply:
commit number 8f08f70

Reply:
@hdamker have pushed fix for this for your review.

Reply:
totally understand. I myself am having a hard time keeping a track of all the updates against each of the review comments. We do the best we can but Its natural to find bugs at a later point in time and we will address them.

while working on the review comments i took an opportunity to think through what i would do as a developer and error response 400 indicating its a bad request felt like a good fit @hdamker .
below example in Generic400 seemed like a good fit for this scenario.
GENERIC_400_INVALID_ARGUMENT:
              description: Invalid Argument. Generic Syntax Exception

I like keeping the response body of GET and request body of PATCH consistent. In my opinion it makes a developer's life easy. In a scenario where there is a new version of the application where the latency threshold has been more relaxed, the developer would simply do a GET, use the response body to modify the minimum latency value and push that as the request body in a PATCH call without having to recreate the object.

I will consider updating to createApplicationProfile as part of another review comment(#79 ) recommending to create a generic applicationProfile object.

Reply:
Feel free to resolve the conversation.

Reply:
thank you @hdamker . Marking the conversation resolved.